template/ios/fastlane/Fastfile

```diff
@@ -39,21 +49,65 @@ platform :ios do
       raise "Invalid environment: #{env}"
     end
     Dotenv.load("../../.env.#{env}")
```

Comment:
remove this line cuz we have lane `before_all`

Reply:
removed